bin/prod

```diff
@@ -0,0 +1,53 @@
+PROD_ECR_REPO="public.ecr.aws/a5f0m6q3/syncm8_prod"
+
+# Build prod image.
+dockerBuild(){
+    bin/util logAndRun "docker build --pull --rm --target prod -t $PROD_ECR_REPO ."
+}
+
+# Push prod image to aws ecr.
+dockerPush(){
+    echo "Are you sure you want to push an image to PRODUCTION?????"
+    read ans
+    if [ $ans == "y" ]; then
+        bin/util logAndRun "docker push $PROD_ECR_REPO"
+    fi
+}
+
+# Restart AWS ECS service.
+awsDeploy(){
+    bin/infra priv "aws ecs update-service --force-new-deployment --service service --cluster backend-cluster | cat"
+}
+
+
+# Print help message
+printHelp(){
+    echo "Usage:  bin/infra COMMAND
+
+Command Runner for syncm8 infra commands.
```

Comment:
qq: would it make sense or difficult to have a command to one-shot a build => push => deploy function?

Reply:
Nah, this is mostly one off debugging use cases. In the happy path, everything is handled by github actions.